docs/source/contributing/developing-core.md

```diff
@@ -54,22 +54,26 @@ For more information about pnpm workspaces, read the [documentation of pnpm work
 ├─ ...
 ```
 
-## Development requirements
+## Development pre-requisites
 
-To set up a Volto core development environment, you need Node.js installed in your system.
-Then, set up the package manager pnpm.
+To set up a Volto core development environment, your system must satisfy the following pre-requisites.
 
-We recommend that you install Node.js using nvm.
-Alternatively you can install Node.js using Homebrew or other package installer.
+```{include} ./install-operating-system.md
+```
+
+-   {term}`nvm`
+-   {term}`Node.js` LTS 20.x
+-   {term}`pnpm`
+-   {term}`GNU make`
 
 
 ### nvm
```

Comment:
@stevepiercy Oh, is that because the NVM_VERSION substitution didn't work that way?

My guess is this is the _only_ place we need that variable, so it would probably make more sense to keep the version inline (i.e. stop using variable substitution for this) so we can include the same file in both sets of install docs.

Reply:
When I upgrade our theme to use Sphinx >=7.2.5, then we can use substitutions in included files. The themes on which we depend added support for Sphinx 7 about two months ago. For now, it's copy-paste.

Reply:
See https://github.com/plone/documentation/issues/1595